abi-bindings/go/teleporter/registry/TeleporterRegistry/packing.go

```diff
@@ -33,6 +33,35 @@ func init() {
 	}
 }
 
+// ProtocolRegistryEntry is currently only packed together with the destinationAddress
+// in the TeleporterRegistryWarpPayload struct. We still implement the ABIPacker interface for exhaustiveness testing.
+
+func (p *ProtocolRegistryEntry) Pack() ([]byte, error) {
+	args := abi.Arguments{
+		{
+			Name: "protocolRegistryEntry",
+			Type: protocolRegistryEntryType,
+		},
+	}
+	return args.Pack(p)
+}
+
+func (p *ProtocolRegistryEntry) Unpack(b []byte) error {
+	args := abi.Arguments{
+		{
+			Name: "protocolRegistryEntry",
+			Type: protocolRegistryEntryType,
+		},
+	}
+	unpacked, err := args.Unpack(b)
+	fmt.Println("unpacked: ", unpacked)
```

Comment:
Let's remove this line

Reply:
Good catch, I removed the line in other files but missed this one.